rfd/0176/README.md

```diff
@@ -120,17 +141,25 @@ blocks under certain circumstances.  Those circumstances will need to change
 and perhaps there will need to be changes in what is written to the various
 boot areas.
 
+XXX KEBE ASKS: Would dedicating a part of a disk or disk pair (say SSDs that
+could be partitioned into ZFS boot and `zones` slog partitions) be an
+alternative to making sure some set of `zones` disks have boot sectors on
+them?
```

Comment:
Regardless of which disks contain boot sectors, something needs to ensure that when they are replaced, the replacements also get the boot sectors.  The zfs resilvering code takes care of data that is within ZFS, but does not do it for the MBR, ESP, or zfs boot area.  Traditionally, pools created with `zpool create -B` will have things set up in a way that disk replacement triggers something along the lines of `bootadm install-bootloader ...`.  The solution we come up with may hook into that without requiring any changes, but I'm not sure about that.

Reply:
Tried to fold in your text.